build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if command -v dpkg-query -l zip
+then
+  mkdir dist
+  zip dist/altapay.zip -r * -x build.sh .gitignore guide.md @
```

Comment:
There seem to be a build task in the ant build.xml. We should use either or.
- The argument for using ant is that it is platform agnostic, meaning any machine could build it as long as they have ant installed.
If the result is the same, I would keep only build.sh instead.

Reply:
I have tested build.sh file and it works fine with windows, mac, and ubuntu machine. I suggest we should keep build.sh file